src/client/editor/SceneLoader.js

```diff
@@ -176,8 +303,15 @@ function inflateGLTFComponents(scene, addComponent) {
       }
     }
 
-    if (object instanceof THREE.Mesh && object.material instanceof THREE.MeshStandardMaterial) {
-      addComponent(object, "standard-material", undefined, true);
+    if (object instanceof THREE.Mesh) {
+      addComponent(object, "mesh", undefined, true);
```

Comment:
We should avoid passing or assigning `undefined`, it gets confusing when you care about whether a method was intentionally called with no value or whether it was never passed in in the first place. Use `null`.

Same for the two `addComponent`s below.